Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nvidia/cuda:11.8.0-base-ubuntu22.04
+
+ENV	NVIDIA_DRIVER_CAPABILITIES=all
+
+# Install required packages
+RUN apt-get update && apt-get install -y \
+    wget \
+    tar \
+    psmisc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download and extract the Titan Node Pool binary
+RUN wget https://github.com/Video-Miner/Releases/releases/download/go-livepeer-0.5.38_videominer-1.2.3-5344efd4/linux-videominer-go-livepeer-0.5.38_videominer-1.2.3-5344efd4.tar.gz && \
```

Comment:
This can be improved so that the latest version is always pulled.

Reply:
This can be done be using the following Dockerfile:

```Dockerfile
FROM nvidia/cuda:11.8.0-base-ubuntu22.04

ENV	NVIDIA_DRIVER_CAPABILITIES=all

# Install required packages
RUN apt-get update && apt-get install -y \
    wget \
    tar \
    psmisc \
    jq \
    && rm -rf /var/lib/apt/lists/*

# Fetch the latest release version from GitHub API
RUN export LATEST_VERSION=$(curl -s https://api.github.com/repos/Livepool-io/transcoder/releases/latest | jq -r .tag_name)

# Download and extract the latest Titan Node pool binary
RUN wget "https://github.com/Livepool-io/transcoder/releases/download/$LATEST_VERSION/${LATEST_VERSION#v}-livepool-linux-amd64.tar.gz" && \
    tar -xzf "${LATEST_VERSION#v}-livepool-linux-amd64.tar.gz" && \
    rm "${LATEST_VERSION#v}-livepool-linux-amd64.tar.gz" && \
    mv "./livepool-linux-amd64/livepool" /usr/local/bin

# Set the entrypoint to start the Titan Node pool binary with command line arguments
ENTRYPOINT ["/usr/local/bin/livepool"]
```

Reply:
I'd like you to put this here for future reference. As stated in the discord:

> Moreover, given your access to the binary source code, you can consider employing a multi-layer build. This approach entails constructing the binary in a dedicated build step and transferring it to a final image, ready for publication on Docker Hub.

This will allow for faster deployment ⚡.

Reply:
hi. How I can run this image in win docker?
![2023-12-07_145857](https://github.com/Video-Miner/Releases/assets/25245789/df9b38bb-101a-46f3-85a6-62c81441c9b7)

Reply:
@PavelKovL I have yet to use docker desktop or run docker on Windows without [WSL](https://learn.microsoft.com/en-us/windows/wsl/install). I, therefore, am not the best person to help you 😅. You, however, should be able to pull and run the container from [dockerhub](https://hub.docker.com/layers/rickstaa/video-miner-pool-miner/latest/images/sha256-9a2a06e6d1a7990f6bca8d347bfd7fff5fea1e1df82bae823232559df66fae73?context=repo) without having to manually build the container locally.